libraries/testing/include/eosio/testing/tester.hpp

```diff
@@ -62,7 +62,7 @@ namespace eosio { namespace testing {
       old_bios_only,
       preactivate_feature_only,
       preactivate_feature_and_new_bios,
-      full
+      complete
```

Comment:
Any particular reason for this renaming?

Reply:
I felt that the name `complete` doesn't imply all protocol features be _fully_ utilized. Do you have another suggestion?

Reply:
I don't understand what you mean.

Reply:
To elaborate a bit, it seems to me that if full implies the features are _fully_ utilized, then complete equally implies that they are _completely_ utilized.

Reply:
Renamed back to the original name of `complete`.